tacs/constraints/adjacency.py

```diff
@@ -97,7 +97,8 @@ def _initializeAdjacencyList(self):
                 elemConn = elemInfo.nodes
                 compID = self.meshLoader.nastranToTACSCompIDDict[elemInfo.pid]
                 nnodes = len(elemConn)
-                if nnodes >= 2:
+                # This checks specifically for adjacency between 2D elements
+                if nnodes > 2:
```

Comment:
Is there a better way we can do this check for 1/2D elements? A beam element could have 3 nodes, and solid elements also have `nnodes > 2`

Reply:
Not sure how usable it is but this pyNastran function returns an `element_dim` value that is exactly what we're looking for here

https://github.com/SteveDoyle2/pyNastran/blob/e468c08b506286eed4b74ec3ab572e1f2d5b1e91/pyNastran/converters/nastran/gui/utils.py#L123